pynitrokey/cli/start.py

```diff
@@ -87,32 +91,70 @@ def set_identity(identity):
 @click.option(
     "--gnuk", default=None, callback=validate_gnuk, help="path to gnuk binary"
 )
-@click.option("-f", "default_password", is_flag=True, default=False,
-    help=f"use default Admin PIN: {DEFAULT_PW3}")
+@click.option(
+    "-f",
+    "default_password",
+    is_flag=True,
+    default=False,
+    help=f"use default Admin PIN: {DEFAULT_PW3}",
+)
 @click.option("-p", "password", help="use provided Admin PIN")
-@click.option("-e", "wait_e", default=DEFAULT_WAIT_FOR_REENUMERATION, type=int,
-    help="time to wait for device to enumerate, after regnual was executed on device")
+@click.option(
+    "-e",
+    "wait_e",
+    default=DEFAULT_WAIT_FOR_REENUMERATION,
+    type=int,
+    help="time to wait for device to enumerate, after regnual was executed on device",
+)
 @click.option("-k", "keyno", default=0, type=int, help="selected key index")
 @click.option("-v", "verbose", default=0, type=int, help="verbosity level")
 @click.option("-y", "yes", default=False, is_flag=True, help="agree to everything")
-@click.option("-b", "skip_bootloader", default=False, is_flag=True,
-    help="Skip bootloader upload (e.g. when done so already)")
 @click.option(
-    "--green-led", is_flag=True, default=False,
-    help="Use firmware for early 'Nitrokey Start' key hardware revisions"
+    "-b",
+    "skip_bootloader",
+    default=False,
+    is_flag=True,
+    help="Skip bootloader upload (e.g. when done so already)",
+)
+@click.option(
+    "--green-led",
+    is_flag=True,
+    default=False,
+    help="Use firmware for early 'Nitrokey Start' key hardware revisions",
 )
-def update(regnual, gnuk, default_password, password, wait_e, keyno, verbose, yes,
-           skip_bootloader, green_led):
+def update(
+    regnual,
+    gnuk,
+    default_password,
```

Comment:
Can we increase line length for such cases? This one decreases readability for no reason

Reply:
As far as I see, we can only increase the total line length (currently 88), but that would not necessarily fix this issue and I would like to keep the maximum line length <= 100.

Reply:
Disabling `black` for the problematic functions triggers a bug:

```
diff --git a/pynitrokey/nethsm/__init__.py b/pynitrokey/nethsm/__init__.py
index f781c2d..6c57852 100644
--- a/pynitrokey/nethsm/__init__.py
+++ b/pynitrokey/nethsm/__init__.py
@@ -648,16 +648,10 @@ class NetHSM:
                 },
             )
 
+    # fmt: off
     def csr(
-        self,
-        country,
-        state_or_province,
-        locality,
-        organization,
-        organizational_unit,
-        common_name,
-        email_address,
-    ):
+            self, country, state_or_province, locality, organization, organizational_unit, common_name, email_address
+    ):  # fmt: on
         from .client.model.distinguished_name import DistinguishedName
 
         body = DistinguishedName(
@@ -676,17 +670,10 @@ class NetHSM:
                 e, state=State.OPERATIONAL, roles=[Role.ADMINISTRATOR]
             )
 
+    # fmt: off
     def key_csr(
-        self,
-        key_id,
-        country,
-        state_or_province,
-        locality,
-        organization,
-        organizational_unit,
-        common_name,
-        email_address,
-    ):
+            self, key_id, country, state_or_province, locality, organization, organizational_unit, common_name, email_address
+    ):  # fmt: on
         data = {
             "countryName": country,
             "stateOrProvinceName": state_or_province,
```

```
error: cannot format pynitrokey/nethsm/__init__.py: INTERNAL ERROR: Black produced code that is not equivalent to the source on pass 1.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /tmp/blk_sxyt0ptu.log
```

Reply:
Let's leave it then